### PR TITLE
[TS] LPS-121719

### PR DESCRIPTION
**Pull request by ChrisKian**
https://issues.liferay.com/browse/LPS-121719

Please let me know if you have any questions, thanks!

### CODE REVIEW

**Comment by ChrisKian** [comment]
ci:test:sf

**Comment by ChrisKian** [comment]
ci:test:relevant

File: modules/apps/frontend-js/frontend-js-svg4everybody-web/src/main/java/com/liferay/frontend/js/svg4everybody/web/internal/servlet/taglib/SVG4EverybodyTopHeadDynamicInclude.java [hunk 1]
```diff
@@ -48,7 +51,24 @@ public void include(
 			HttpServletResponse httpServletResponse, String key)
 		throws IOException {
 
-		if (!PropsValues.CDN_DYNAMIC_RESOURCES_ENABLED ||
+		boolean cdnDynamicResourcesEnabled;
+
+		try {
+			cdnDynamicResourcesEnabled = _portal.isCDNDynamicResourcesEnabled(
```
Comment (shuyangzhou) [inline modules/apps/frontend-js/frontend-js-svg4everybody-web/src/main/java/com/liferay/frontend/js/svg4everybody/web/internal/servlet/taglib/SVG4EverybodyTopHeadDynamicInclude.java:57]: @ChrisKian Isn't the Portal.isCDNDynamicResourcesEnabled() already fallback to PropsValues.CDN_DYNAMIC_RESOURCES_ENABLED ? Why do you need to do this explicit fallback?

File: modules/apps/frontend-js/frontend-js-svg4everybody-web/build.gradle [hunk 2]
```diff
@@ -1,21 +1,19 @@
...
-
 	filter {
 		String line ->
 
 		line.replace "\"function\" == typeof define", "false && \"function\" == typeof define"
 	}
 
-	from npmInstall.nodeModulesDir
-	include "svg4everybody/dist/svg4everybody.js"
+	from FileUtil.get(project, "https://raw.githubusercontent.com/jonathantneal/svg4everybody/d6ea438bd5a4599c35cbf1a0c29d697569c8c6c6/dist/svg4everybody.js")
```
Comment (shuyangzhou) [inline modules/apps/frontend-js/frontend-js-svg4everybody-web/build.gradle:15]: @ChrisKian this just look too weird to me. You should send this to @petershin

**Comment by shuyangzhou** [comment]
@ChrisKian see the comments above, I think you don't need that explicit fallback logic, just use the Portal.isCDNDynamicResourcesEnabled() should be good enough, and please send this to @petershin to work out that gradle download thing.

**Comment by ChrisKian** [comment]
Sent to https://github.com/petershin/liferay-portal/pull/1258

**Pull request closed without merging**